news/2024/2024-09-17-performance-points-star-rating-updates.md

```diff
@@ -0,0 +1,139 @@
+---
+layout: post
+title: Performance Points & Star Rating Updates
+date: 2024-09-17 12:00 +0000
+---
+
+The time for change is finally here!
+
+![](/wiki/shared/news/banners/star-rating.jpg)
```

Comment:
since last time, new banner designs have been made https://github.com/ppy/osu-wiki/blob/master/wiki/shared/news/banners/pp-sr-osu.jpg

```suggestion
![](/wiki/shared/news/banners/pp-sr-osu.jpg)
```